packages/t/tabler-icons.json

```
@@ -19,20 +19,17 @@
   },
   "autoupdate": {
     "source": "npm",
-    "target": "tabler-icons",
+    "target": "@tabler/icons-webfont",
     "fileMap": [
       {
-        "basePath": "",
+        "basePath": "dist",
         "files": [
-          "tabler-*.svg",
-          "icons/*.svg",
-          "iconfont/tabler-*.@(css|scss)",
-          "iconfont/fonts/tabler-*"
+          "**/!(*.html|*.scss)"
```

```suggestion
          "fonts/*.@(ttf|woff|woff2)",
          "*.css"
```
Please avoid using globstars as they can result in files being added to the CDN that we don't want.